src/modules/menu/components/settings/MenuSettings.vue

```diff
@@ -11,5 +18,6 @@ import LinksToolbar from '@/modules/menu/components/settings/LinksToolbar.vue'
         <div id="menu-links">
             <LinksToolbar />
         </div>
+        <MapFooterAppVersion v-if="isPhoneMode" />
```

Comment:
I would not place the version on mobile PROD here, it is too predominant ! I know this is helpful for developers but not for the users. You can maybe try to keep it in the footer on mobile in the center very small and in white that we almost don't see it.

Reply:
I made it way smaller, and not take more space than before (position absolute)